dashboard/app/models/script_level.rb

```diff
@@ -30,9 +30,10 @@ class ScriptLevel < ActiveRecord::Base
   include SharedConstants
   include Rails.application.routes.url_helpers
 
+  belongs_to :script
+  belongs_to :lesson_group
```

Comment:
is this used / does it work? if it does it would help me to understand what it does. I thought there needed to be a lesson_group_id field for this to work. it might be possible to do:
```
has_one: :lesson_group, through: lesson
```
https://guides.rubyonrails.org/association_basics.html#the-has-one-through-association